packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Text } from '@deriv/components';
+import { MT5AccountStatus } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 
 const getStatusBadgeConfig = (
-    account_status: string,
-    openFailedVerificationModal: (selected_account_type: string) => void,
-    selected_account_type: string
+    account_status: string | undefined | null,
```

Comment:
```suggestion
    account_status: typeof MT5AccountStatus[keyof typeof MT5AccountStatus] | 'pending' | 'failed' | 'need_verification',
```
Should be typing this from the import. Although the account status  'pending' | 'failed' | 'need_verification' should be coming from the api